modules/installation-aws-config-yaml-customizations.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/ipi/installing-aws-customizations.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="installation-aws-config-yaml-customizations_{context}"]
+= Sample customized install-config.yaml file for AWS
+
+You can customize the installation configuration file (`install-config.yaml`) to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+For a full list and description of all installation configuration parameters, see the page titled _Installation configuration parameters_.
+====
+
+[source,yaml]
+----
+apiVersion: v1 <1>
+baseDomain: example.com
+sshKey: ssh-ed25519 AAAA...
+pullSecret: '{"auths": ...}'
+metadata:
+  name: example-cluster
+controlPlane: <2>
+  name: master
+  platform:
+    aws:
+      type: m6i.xlarge
+  replicas: 3
+compute: <3>
+-  name: worker
+  platform:
+    aws:
+      type: c5.4xlarge
+  replicas: 3
+networking: <4>
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+platform: <5>
+  aws:
+    region: us-west-2
+----
+<1> Parameters at the first level of indentation apply to the cluster globally.
+<2> The `controlPlane` stanza applies to control plane machines.
+<3> The `compute` stanza applies to compute machines.
+<4> The `networking` stanza controls the cluster networking configuration. If networking values are not provided, the installation program provides default values.
+<5> The `platform` stanza controls aspects of the infrastructure platform that hosts the cluster.
```

Comment:
Not sure if there is a semantic difference intended between "applies to" and "controls" - would it be possible to use one term across all of them?